firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidator.java

```diff
@@ -104,6 +109,17 @@ private boolean areCountersValid(@Nullable TraceMetric trace, int deep) {
     return true;
   }
 
+  private boolean isValidScreenTrace(@NonNull TraceMetric trace) {
```

Comment:
Can you move`isValidScreenTrace` to`isValidTrace` for clarity? because sreenTrace is a subset of traceMetric.